website/docs/code-splitting/local-vs-remote-chunks.md

```diff
@@ -0,0 +1,307 @@
+meaning `src_Button_js` chunk in production might look like `137`.
+
+This behavior is fine for Web, but in React Native with `ScriptManager`, it's not ideal, because we don't know what
+this number will be in production.
+
+There are 2 options to address this problem:
+
+1. Set [`optimization.chunkIds`](https://webpack.js.org/configuration/optimization/#optimizationchunkids) option to `named` in Webpack config.
+1. Use `/* webpackChunkName: "<name>" */` magic comment in `import(...)`.
+
+:::tip
+
+You can use both `optimization.chunkIds` and `webpackChunkName` comment at the same time. They are not mutually exclusive.
+
+:::
+
+:::tip
+
+Chunk extension can be configured in [`output.chunkFilename`](https://webpack.js.org/configuration/output/#outputchunkfilename), which is set to `.chunk.bundle` by default.
+It's usually __not__ necessary to modify it.
+
+:::
+
+
+### Named `chunkIds` config option
+
+Setting [`optimization.chunkIds`](https://webpack.js.org/configuration/optimization/#optimizationchunkids) option to `named` in your Webpack config will
+force Webpack to always use the human-readable form for the name of the chunks.
+
+:::info
+
+In newer versions of Re.Pack's templates for Webpack config, `chunkIds` is set to `named` by default.
+
+:::
+
+```js
+/* ... */
+
+export default (env) => {
+  /* ... */
+
+  return {
+    /* ... */
+
+    optimization: {
+      /* ... */
+
+      chunkIds: 'named',
+    },
+
+    /* ... */
+  };
+}
+```
+
+:::tip
+
+Keep in mind that, `webpackChunkName` magic comment will always take precedence, so even if you have `chunkIds: 'named'`, Webpack will use name in `webpackChunkName`
+comment for that chunk instead of the inferred one.
+
+:::
+
+### `webpackChunkName` magic comment
+
+`webpackChunkName` magic comment allows you to provide your own custom name that should be used for the chunk when calling `import(...)` function:
+
+```js
+import(/* webpackChunkName: "button" */ './Button');
+```
+
+This example will result in chunk being named `button`, so the filename with extension will be `button.chunk.bundle`.
+
+:::tip
+
+You can use `webpackChunkName` magic comment to provide your custom name, regardless of the `optimization.chunkIds` option.
+`webpackChunkName` will always take precedence.
+
+:::
+
+## Remote chunks
+
+By default all chunks are remote chunks, meaning they are not bundled into the application and will be downloaded from the remote location (usually the Internet) on demand.
+This helps with reducing the initial application size, especially if you have logic or features that only a subset of users will use - it doesn't make sense for everyone else
+to always have to download the code (together with the application) they won't need.
+
+All remote chunks are stored under `<projectRoot>/build/output/<platform>/remotes` by default, for example if `button.chunk.bundle` is a remote chunk, it will be stored under:
```

Comment:
```suggestion
All remote chunks are stored under `<projectRoot>/build/output/<platform>/remotes` by default. For example if `button.chunk.bundle` is a remote chunk, it will be stored under:
```